c2corg_ui/__init__.py

```diff
@@ -28,6 +28,8 @@ def main(global_config, **settings):
     config.add_route('waypoints_view', '/waypoints/{id}/{lang}')
     config.add_route('waypoints_index_default', '/waypoints')
     config.add_route('waypoints_index', '/waypoints/list*filters')
```

Comment:
Is the "remainder match" (`*filters`) required? Because normal query parameters (`?...`) are used, I think you don't need it.